src/sqlite3pp.cpp

```diff
@@ -75,6 +75,31 @@ namespace sqlite3pp
     }
   }
 
+  database::database(database&& db)
+    : db_(std::move(db.db_))
```

Comment:
Please use the following style.

database::database(database&& db) : db_(std::move(db.db_)),
    bh_(std::move(db.bh_)),
    ch_(std::move(db.ch_)),
    ...